src/wasm-lib/kcl/src/modules.rs

```diff
@@ -119,7 +119,7 @@ pub enum ModuleRepr {
 }
 
 #[allow(clippy::large_enum_variant)]
-#[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize, Hash)]
+#[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize, Hash, ts_rs::TS)]
 pub enum ModulePath {
```

Comment:
I would add this so that it's easier to do type narrowing in TS. I don't think this PR adds using it from TS yet, right? But we'll want this.

```suggestion
#[serde(tag = "type")]
pub enum ModulePath {
```

Reply:
Added.